src/layer/LayersFactory.ts

```diff
@@ -109,17 +115,18 @@ export class LayersFactory {
   public static async makeLayers(
     baseUrl: string,
     filterLayers: Function | null = null,
+    overrideSHLayerParams?: Record<string, any>,
```

Comment:
I don't think this is the most accurate name for this variable as you're setting default constructor parameters here and not overriding some values from service. Perhaps `defaultConstructorParameters` or something would be better? Although this is just semantic nitcpicking as intend is clear enaugh. fftr.

Reply:
Me and @sinergise-anze came up with this name after discussing the problem, so I would leave it as is.

Reply:
:+1:

Reply:
Actually, we came up with `overrideLayerParams` which is something else entirely... :smile: 

To me `override` seems fitting, because we are overriding the values we get from the layer definition on the service. `overrideConstructorParams` maybe?

Reply:
True, my bad, original was `overrideLayerParams`, I just added `SH` since currently only legacy functions use that and only for SH layers.

To me `overrideSomething` seems ok, since the layer classes default behaviour is to use the values from the services (and it seems that this wasn't obvious before). But I don't care much what comes after that. Will change to `overrideConstructorParams`.

Reply:
Was changed